pkg/cmd/version/version.go

```diff
@@ -68,24 +70,51 @@ func Command(p cli.Params) *cobra.Command {
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			fmt.Fprintf(cmd.OutOrStdout(), "Client version: %s\n", clientVersion)
+			if component == "" {
+				fmt.Fprintf(cmd.OutOrStdout(), "Client version: %s\n", clientVersion)
```

Comment:
Can we include it in switch case of `""`

Reply:
@piyush-garg , "" would be invalid right .. and it should be covered under default case. What do you think?

Reply:
No No, i dont mean like component = `""`, my comment is for if component value is empty, then why to have a if case for client version here, we can print it in case `""` only

Reply:
Ohh .. okay .. I will include that in switch case ""